package.json

```diff
@@ -119,6 +121,7 @@
     "nsp": "nsp check",
     "pretest": "npm run hint",
     "test": "npm run test:unit && npm run coverage",
+    "test:unitlocal": "mocha --compilers js:babel-core/register --require babel-polyfill --reporter-options mochaFile=$(bash scripts/test_output_dir.sh junit)/junit.xml test",
```

Comment:
can we use `test:unit:dev` to match `npm run db:migrate:dev` to apply to local dev environment?